src/projects/projects-page.html

```diff
@@ -30,6 +30,20 @@
         z-index: 2;
         height: 100%;
       }
+      .overlay-shadow {
+        transition: all 0.4s ease;
+        opacity: 0;
+        background-color: rgba(0,0,0,1);
+        position: fixed;
+        width: 100%;
+        height: 100%;
+        pointer-events: none;
+        z-index: 1;
+      }
+      .collapsable.both-list ~ .overlay-shadow {
```

Comment:
`+` is for direct siblings, `~` is for any sibling. Since `.overlay-shadow` is not directly next to `.collapsable`, the `+`-selector would not work.